lib/web/apiserver.go

```diff
@@ -125,6 +125,9 @@ const (
 	IncludedResourceModeRequestable = "requestable"
 	// IncludedResourceModeAll describes that all resources, requestable and available, should be returned.
 	IncludedResourceModeAll = "all"
+	// DefaultLicenseWatchInterval is the default time in which the license watcher
+	// should ping the auth server for new cluster features
```

Comment:
```suggestion
	// should ping the auth server to check for an updated license.
```

Reply:
Updated to `should ping the auth server to check for updated features` as per discussion below about features vs license.